### ISSUE TITLE
Invert direction when controlling slider using mouse wheel

### ISSUE THREAD

**Issue by bitrevo** [emoacht/Monitorian#413]
Currently we can hover the mouse cursor on the brightness slider and adjust the value using mouse wheel.
- Monitorian mouse wheel up -- brightness decrease
- Monitorian mouse wheel down -- brightness increase

It is working fine but it will be better the behavior can match Windows 11 OS, for example to adjust the sound volume in Windows action center.
- Windows mouse wheel up -- sound volume increase
- Windows mouse wheel down -- sound volume decrease

**Comment by emoacht** [comment]
First of all, it should be clear what up/down directions mean for mouse wheel.

I made this image for this purpose.
![mouse-wheel-direction](https://user-images.githubusercontent.com/7205690/218227516-33d1636a-72a8-4853-ac84-ed93d3d7b989.png)

Then, I designed to reflect the input from mouse wheel to a slider as following:

- UP -> Increase slider value
- DOWN -> Decrease slider value

I am not sure what happens in your particular case.
